js/source/video_source.js

```diff
@@ -107,92 +97,18 @@ VideoSource.prototype = util.inherit(Evented, /** @lends VideoSource.prototype *
     /**
      * Sets the video's coordinates and re-renders the map.
      *
+     * @method setCoordinates
      * @param {Array<Array<number>>} coordinates Four geographical coordinates,
      *   represented as arrays of longitude and latitude numbers, which define the corners of the video.
      *   The coordinates start at the top left corner of the video and proceed in clockwise order.
      *   They do not have to represent a rectangle.
      * @returns {VideoSource} this
      */
```

Comment:
Could we eliminate the need for this inherited comment by rewording it? `s/video/source` or `s/video/data` or `s/video/layer`?